src/stores/global-store.ts

```diff
@@ -3,8 +3,13 @@ import { defineStore } from 'pinia'
 export const useGlobalStore = defineStore('global', {
   state: () => {
     return {
-      isSidebarMinimized: false,
       userName: 'Vasili S',
+      email: 'vasili@gmail.com',
```

Comment:
Let's move user name, member since, profile picture, is2FA enabled to `userStore`.